### PR TITLE
added courses on blockchain and bash

### PR DESCRIPTION
**Pull request by riyadhami**
## What does this PR do?
Add new courses 

### Description
Added a youtube course on beginners intro to bash and a course on blockchain using javascript .

### Why is this valuable (or not)?
These courses helped me start out and i think would be good resources for other beginners.

### How do we know it's really free?
It's on youtube.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -240,6 +241,7 @@
 * [Build 5 Dapps on the Ethereum Blockchain - Beginner Tutorial](https://www.youtube.com/watch?v=8wMKq7HvbKw) - Julien Klepatch, EatTheBlocks (freeCodeCamp.org)
 * [Build and Deploy Your First Decentralized App with Etherem](https://www.udemy.com/course/your-first-decentralized-app/) - Gary Simon (Udemy)
 * [CryptoZombies.io](https://cryptozombies.io) - CleverFlare, Loom Network
+* [Learn Blockchain, Solidity, and Full Stack Web3 Development with JavaScript – 32-Hour Course](https://www.youtube.com/watch?v=gyMwXuJrbJQ&t=1881s) - Patrick Collins (freeCodeCamp.org)
```
Comment (eshellman) [inline courses/free-courses-en.md:244]:
```suggestion
* [Learn Blockchain, Solidity, and Full Stack Web3 Development with JavaScript – 32-Hour Course](https://www.youtube.com/watch?v=gyMwXuJrbJQ&t=1881s) - Patrick Collins (freeCodeCamp)
```

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -223,6 +223,7 @@
 
 * [Bash Basics for Cloud Computing](https://www.udemy.com/course/bash-basics-for-cloud-computing/) - Kumulus Technologies (Udemy)
 * [Bash Scripting Full Course 3 Hours](https://www.youtube.com/watch?v=e7BufAVwDiM) - Linuxhint
+* [Beginner's Guide to the Bash Terminal](https://www.youtube.com/watch?v=oxuRxtrO2Ag&t=569s) - Joe Collins (YouTube)
```
Comment (eshellman) [inline courses/free-courses-en.md:226]:
```suggestion
* [Beginner's Guide to the Bash Terminal](https://www.youtube.com/watch?v=oxuRxtrO2Ag&t=569s) - Joe Collins
```

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -223,6 +223,7 @@
 
 * [Bash Basics for Cloud Computing](https://www.udemy.com/course/bash-basics-for-cloud-computing/) - Kumulus Technologies (Udemy)
 * [Bash Scripting Full Course 3 Hours](https://www.youtube.com/watch?v=e7BufAVwDiM) - Linuxhint
+* [Beginner's Guide to the Bash Terminal](https://www.youtube.com/watch?v=oxuRxtrO2Ag&t=569s) - Joe Collins
```
Comment (shrn01) [inline courses/free-courses-en.md:226]: Please remove the timestamp from the link

Comment (riyadhami) [reply courses/free-courses-en.md]: removed the time stamps. Thanks for reviewing !

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -223,6 +223,7 @@
 
 * [Bash Basics for Cloud Computing](https://www.udemy.com/course/bash-basics-for-cloud-computing/) - Kumulus Technologies (Udemy)
 * [Bash Scripting Full Course 3 Hours](https://www.youtube.com/watch?v=e7BufAVwDiM) - Linuxhint
+* [Beginner's Guide to the Bash Terminal](https://www.youtube.com/watch?v=oxuRxtrO2Ag) - Joe Collins (YouTube)
```
Comment (shrn01) [inline courses/free-courses-en.md:226]:
```suggestion
* [Beginner's Guide to the Bash Terminal](https://www.youtube.com/watch?v=oxuRxtrO2Ag) - Joe Collins
```

**Pull request closed without merging**